### PR TITLE
Get guests api

### PR DESCRIPTION
**Pull request by fieldstyler**
# Pull Request

## What's this PR do?

- [x] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Testing
- [ ] Styling
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Description of changes?

Add three new endpoints:
 - POST api/v1/weddings/create
 - POST api/v1/weddings/create_guest
 - GET api/v1/weddings/guests

Delete Guest app and add all other functionality into Wedding app

## What are the relevant tickets?
* Addresses
* Closes

### CODE REVIEW

File: app/wedding/serializers.py [hunk 1]
```diff
@@ -1,8 +1,15 @@
...
 class WeddingSerializer(serializers.ModelSerializer):
     class Meta:
         model = Wedding
         fields = ('id', 'name', 'email', 'date', 'image')
+
+
+class GuestSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Guest
+        fields = ('id', 'name', 'phone_number', 'wedding')
```
Comment (IamNorma) [inline app/wedding/serializers.py:15]: Great work with the serializers in this file. And good to know that wedding needed to be included in the fields in order to complete the connection between wedding and guest

File: app/wedding/tests/test_guest_api.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+from rest_framework import status
+
+from core.models import Wedding
+from core.models import Guest
+
+from wedding.serializers import GuestSerializer
+
+CREATE_WEDDING_URL = reverse('wedding:create')
+GUESTS_URL = reverse('wedding:guest-list')
+CREATE_GUEST_URL = reverse('wedding:create_guest')
```
Comment (IamNorma) [inline app/wedding/tests/test_guest_api.py:14]: If we want more customized urls that follow the convention we're used to, we can probably look into getting rid of reverse.

File: app/wedding/views.py [hunk 3]
```diff
@@ -1,11 +1,31 @@
...
+
+class GuestViewSet(viewsets.ModelViewSet):
+    """Manage guests in the database"""
+    queryset = Guest.objects.all()
+    serializer_class = serializers.GuestSerializer
+
+
+class CreateGuestView(generics.CreateAPIView):
+    """Create a new guest in the system"""
+    serializer_class = GuestSerializer
```
Comment (IamNorma) [inline app/wedding/views.py:31]: These views look good to me. Nice work

**Review by IamNorma — APPROVED** [review]

File: app/wedding/serializers.py (hunk 1 above)
Comment (fieldstyler) [reply app/wedding/serializers.py:15]: What an annoying thing to miss for 24 hours! :P

File: app/wedding/tests/test_guest_api.py (hunk 2 above)
Comment (fieldstyler) [reply app/wedding/tests/test_guest_api.py:14]: Great point! We should definitely look into that!

**Pull request merged**